ksqldb-engine/src/main/java/io/confluent/ksql/engine/rewrite/DataSourceExtractor.java

```diff
@@ -131,6 +134,42 @@ public Void visitAliasedRelation(final AliasedRelation relation, final Void ctx)
       return null;
     }
 
+    private String checkAlternatives(final SourceName sourceName) {
```

Comment:
We should probably give hints in other commands where it makes sense:

```sql
ksql> drop stream s1;
Stream S1 does not exist.
```

Reply:
> We should probably give hints in other commands where it makes sense. For example:
> 
> ```sql
> ksql> drop stream s1;
> Stream S1 does not exist.
> ```

This needs to be addressed in another PR.